MomentumDiscordBot/Bot.cs

```diff
@@ -24,7 +25,7 @@ public Bot(Configuration config, ILogger logger)
 
             var logFactory = new LoggerFactory().AddSerilog(logger);
 
-            _discordClient = new DiscordClient(new DiscordConfiguration
+            _discordClient = new DiscordShardedClient(new DiscordConfiguration
```

Comment:
Why sharded clients? Is this a requirement of slash commands?

Reply:
ShardedClients helps to make the bot more stable when under intense load. It does so by dividing resources.